<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1754</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customPlot_1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_2" native="true">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_3" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_4" native="true">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_avgSig" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>611</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_sig_m1" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>611</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_sig_m2" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>440</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_img_count" native="true">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>10</y>
      <width>461</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1754</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
